app/javascript/components/patient/monitoring_actions/CaseStatus.js

```diff
@@ -55,9 +64,13 @@ class CaseStatus extends React.Component {
         !confirmedOrProbable &&
         (this.props.patient.case_status === 'Confirmed' || this.props.patient.case_status === 'Probable')
       ) {
+        let modal_text = `This case will be moved to the exposure workflow and will be placed in the symptomatic, non-reporting, or asymptomatic line list as appropriate to continue exposure monitoring.`;
```

Comment:
I think this can just use `''` instead of ``

Reply:
[c8eeb6b](https://github.com/SaraAlert/SaraAlert/pull/1242/commits/c8eeb6b110cd2a5877f4e425c71db324cffa7531)